src/openMVG/rig/Rig.cpp

```diff
@@ -60,17 +60,16 @@ bool Rig::initializeCalibration()
   // check that there are cameras
   assert(_vLocalizationResults.size()>0);
   
-  // check that all the cameras have the same number of localizationResults
-  const std::size_t sequenceLength = _vLocalizationResults[0].size();
+  // make all the cameras have the same number of localizationResults (equal to the shortest clip)
   const std::size_t nCams = _vLocalizationResults.size();
-  
-  for(std::size_t i = 1; i < nCams; ++i)
   {
-    assert(_vLocalizationResults[i].size() == sequenceLength 
-            && "All cameras must have the same number of images");
+    std::size_t shortestSeqLength(-1);
+    for (const auto& v: _vLocalizationResults)
+      shortestSeqLength = std::min(shortestSeqLength, v.second.size());
+    for (auto& v: _vLocalizationResults)
```

Comment:
We can stop here if `shortestSeqLength == 0` (yeah, pretty dumb but better safe than sorry :-)

So something like
```c++
if(shortestSeqLength == 0)
{
  POPART_COUT("The calibration results are empty!");
  return false;
}
```